packages/react-bloc/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2017 ericktorres <ericktorresdeveloper@gmail.com>
```

Comment:
2017 👀

Reply:
Nice catch. Should I also change the email and username to yours since the repo is under your account? This was autogenerated and I missed it.

Reply:
Yeah it might be better to keep it consistent